bsp/esp32_s3_usb_otg/include/bsp/esp32_s3_usb_otg.h

```diff
@@ -44,14 +61,22 @@ typedef enum bsp_led_t {
 #define BSP_SD_CMD            (GPIO_NUM_35)
 #define BSP_SD_CLK            (GPIO_NUM_36)
 
+/* Buttons */
+#define BSP_BUTTON_OK_IO            (GPIO_NUM_0)
+#define BSP_BUTTON_DW_IO            (GPIO_NUM_11)
+#define BSP_BUTTON_UP_IO            (GPIO_NUM_10)
+#define BSP_BUTTON_MENU_IO          (GPIO_NUM_14)
+#define BSP_USB_OVERCURRENT_IO      (GPIO_NUM_21)
+
 /* Button */
 // All signal are active low
 typedef enum {
-    BSP_BUTTON_OK = GPIO_NUM_0, // Also serves as BOOT select
-    BSP_BUTTON_DW = GPIO_NUM_11,
-    BSP_BUTTON_UP = GPIO_NUM_10,
-    BSP_BUTTON_MENU = GPIO_NUM_14,
-    BSP_USB_OVERCURRENT = GPIO_NUM_21 // This is not a real button, but the button API can be reused here
+    BSP_BUTTON_OK, // Also serves as BOOT select
+    BSP_BUTTON_DW,
+    BSP_BUTTON_UP,
+    BSP_BUTTON_MENU,
+    BSP_USB_OVERCURRENT, // This is not a real button, but the button API can be reused here
+    BSP_BUTTON_NUM
```

Comment:
I think, that we removed in all BSPs. Do you mean, that somebody used these constants as GPIO numbers in their project? The old BSP API was changed for the new usage.

Reply:
> Do you mean, that somebody used these constants as GPIO numbers in their project?

Yes, that is what I meant. If possible, it would be nice not to change this.

Otherwise LGTM!

Reply:
ok, I changed it.